pkg/reporter/htmlreporter/client.go

```diff
@@ -0,0 +1,36 @@
+package htmlreporter
```

Comment:
There is no use for this file. And it contains commented code and stuff like blabla